### ISSUE TITLE
error: EACCES, mkdir

### ISSUE THREAD

**Issue by mgifford** [keybase/keybase-issues#1491]
I get this with when trying to login.

$ keybase login
error: EACCES, mkdir '/home/mgifford/.local/share/keybase/keybase.idb/8d'

**Comment by zQueal** [comment]
`EACCES` is likely a permission error. Are you running the command as the `mgifford` user, or as root? If you're running as `mgifford` try running `sudo keybase login`.

**Comment by mgifford** [comment]
Yup, that worked..  I had to sudo on all the commands I think...  Wasn't expecting that.

**Comment by zQueal** [comment]
I believe this happens when you install `keybase-installer` as the root user. If you change the permissions for `/home/mgifford/.local/share/keybase` then you shouldn't have to do `sudo` every time.

    sudo chown -R mgilford:mgilford /home/mgifford/.local/share/keybase

**Comment by mgifford** [comment]
Might be good to add some more documetation to this process.
